ironfish/src/networkDefinitions.ts

```diff
@@ -0,0 +1,123 @@
+        },
+        {
+            "type": "Buffer",
+            "data": "base64:AAAAAAAAAAABAAAAAAAAAACA8UsfFPH/AAAAAKrYsEofZEDCEHX7xlLHu9xHFNovc0f5yTAjxnSutZ8XhDXMLgSg1nLtIH4W9HsnVYb1uxB+8F1hyJIb1WxOZhJXRz89l/xYlY0+GtDlHQQIOGPYz5AdWMzVq6HTWdC72Aje9c46bg5ouYNCRr/1mLKrzmSMykbj/rFFAf7xHn8cXipNh6Bn9tXr2c8/9KV+saKetuv5oxxv4b2ugve+n0rBT4M5X7x/eK/n/SJ4WKEy0u56i+bIWM37ldanVGdy/QY+NBeXIh3M9uyjA2P5jqQ7jYGlcY9vW8V7ok4OQcMfJ+hgLLyiEdKxAi1ZbcHA0o8Vei1PQvRQlZZv5RyeUy22NY/rgxh68toXm72QRXoAgLVs7PNufj4FWGPY1DrZPa1XhgCBBOs6yZtAaUhgRWFbw+XScSFv3rsd/bc9nlq6SYSOhGRvHd1/Hckv4jbsP9OAVuBTFNOxvyemXaEiqd3iOTVT7sZbKIEat4/TjR8GCixG9WASuK2ZylMeE2mwc2wGI0JlYW5zdGFsayBub3RlIGVuY3J5cHRpb24gbWluZXIga2V5MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwVtma3xejy1EzEbbB20lLj6EMLa3kWaHvxb4QcU4FwVTomaELUeeceLnoBSzHsi06PvF90jI30lkzREQ1T2pzAQ=="
+        },
+        {
+            "type": "Buffer",
+            "data": "base64:AQAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAJJ+dYi9mGsc+th9Bzd77jeFFYA/TK0AOYPNLpJkqgxOBrLNHSw87drpAh4DRp+4Vpb97/WclXaPLwiAUmutWTK/yVSB3plRXFeQbKre4Td+BfN8eLsEUYkyD/DyLXk2cxTAuzF2K7Uucs3TdTt5ylr9snuDfmHRuphsr9sPLPJC6Q7LJalVbnhTyjtIxdCxH4baKBc5DAWmzMyDnZ2LkPVm+yMI/yA2tkDi+70oSXfNpFJhThutfogiXzTv8eMDe05mfKhGcIcNa8Wa4veP6LejuzKpAJhlYQY+jHjtPqWIH7gVaikGHORDhK9j5GiAZFAx/dvOa99EqeJDqqHEPBT+ggyHzpzhN9xrgpmWgP9JPDw+8txIy/EVrpMMqp51bwIAAACbJIF+WrTsx8my6Hg9q14G4pyf/bYTdhRsWJn2G82bmfW06HTWYUZQSFarUB0y/0FYh5+SbukcAy5zYYxJB76dJORNLw/uw4hJeun72IabpkC+BJ2AgkWBY+QjcP+wEAKgpVj/nlJ66mCsdtQp5byb8p9kH9I3gc/0TgsGBxXoEfXrR2g8mSWGtEl6MNTYofakD1smntaNlAWK6PdhFbzqqj7/Qd158xpwi8qp+nCMDH63ud6lGGStbdZD0LXZzaABW+zp5EZ8Mi+ZOSd5O8BF+/b1dEXyUfYTbdsZeaa+u1qES/mDotKbGwP5E/JzzOiVIfCpebaw59lOL6z/1O4EajXUigyMuwoPCkynbz/yWL+QCyL5NMuuj/eAAfW6lGZdikSIa89vi8j9JKdwX2UT3vTDQ+4q8sqtgCWMABvtyPhCCl2O2q9k+VBDSxIEIwo2QdFvkUDmsjRA4e7mciJP279Q31PioNg/qBUmdRhXNyFFM9gawcUr1w7YWWsOL8lt2CaX6jWEIp+5zsnAuo2Q6ENFCPhpXbPjcOquL5DlnIIXMRqM43aPFlEGT6NfGOVD/BS7I2aLVq1qla71oLmNBlYZ2lZtl3Cf6Ge+UtnnmiLaOulePy7NahBwAGNgECnO9WdkBJODBURLZ9CaeScn5Fe4EsuKMYDnnPrBo1/tfSToA3Di9P4UjABfopb1DTbIamEwplXhHeje/h6OZvf1QLxby23Y7D5mY4Q7seFOXbqgvTtn7thkvDVb212gM66+Ul9bGyv+bIV5Luj0LF+kexFXRkpsXWN9SRQ7Ka3rsliWqYj7eGd2NLwwpCL2nI/eYgo="
+        }
+        ]
+    },
+    "consensus": {
+        "genesisBlockPrevious": {
```

Comment:
Same comment as on the genesis block sequence -- we could probably just make this a constant in the code, since I don't see a reason it would need to be configured, but not necessary for this PR.

Reply:
Made into a constant